src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/ImplementIDisposableCorrectly.cs

```diff
@@ -685,7 +686,8 @@ private bool ValidateOperation(IOperation operation)
                     case OperationKind.Try:
                         return ValidateTryOperation((ITryOperation)operation);
                     default:
-                        return false;
+                        // Ignore attribute operation blocks.
+                        return operation.IsAttribute();
```

Comment:
comment seems wrong? code looks like "don't ignore attribute operation blocks"

Reply:
The validator returns true for ignore/valid operations.